images/image.mk

```diff
@@ -17,6 +17,8 @@
 
 override GOOS=linux
 
+DOCKERCMD := docker
```

Comment:
should this be `DOCKERCMD ?= docker`

Reply:
Generally I don't think that would be necessary because these variables would only be [overridden on the command line](https://www.gnu.org/software/make/manual/html_node/Overriding.html#Overriding) to match the user's environment. However, if we thought the make files could get more complicated with nested includes it might pay off. I'm sure it works OK for the simple use case because that's how I tested it. But, I'm not opposed to the change, do you feel strongly enough about it to have me make the change?

Reply:
I don't feel strongly about it :-)